app/env_canada.py

```diff
@@ -0,0 +1,78 @@
+""" A script that downloads weather models from Environment Canada HTTP data server
+"""
+
+import os
+from pathlib import Path
+from datetime import datetime
+import logging
+import time
+import pytz
+import wget
+
+# pylint: disable=invalid-name, broad-except
+
+
+# Cron job schedule times are based on the timezone of the master where the job is initiated.
+tz = pytz.timezone('America/Vancouver')
+logger = logging.getLogger()
+logging.Formatter.converter = lambda *args: datetime.now(tz=tz).timetuple()
```

Comment:
Can we rather use logging.json - would prefer to if we kept the configuration in a config file.